app/components/DepositWithdraw/blocktrades/BlockTradesBridgeDepositRequest.jsx

```diff
@@ -434,6 +436,25 @@ class BlockTradesBridgeDepositRequest extends React.Component {
             url: checkUrl
         });
 
+        let announcements_url = checkUrl + "/announcements/enabled/trade";
+        let announcements_promise = fetch(announcements_url,
+                                        {method: 'get', headers: new Headers({"Accept": "application/json"})})
+                                    .then(response => response.json());
+
+        Promise.resolve(announcements_promise).then((result) => {
+            this.state.announcements = result;
```

Comment:
Don't mutate `this.state` directly like this, instead use `this.setState({announcements: result})`, and sort results before assigning the result to the state. Same goes for the catch, use this.setState

Reply:
Please verify: https://github.com/bitshares/bitshares-ui/pull/924/commits/42ab23d234e30bcffd986db635f60aa85e470437